src/Pages/jinahhong/Login/LoginH.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./LoginH.scss";
+
+class LoginH extends React.Component {
+  constructor() {
+  super()
+    this.state = {
+      idInputValue: "",
+      pwInputValue: "",
+      backColor: ""
+    }
+  }
+  handleIdInput = (event) => {
+    this.setState({
+      idInputValue: `${event.target.value}`,
+    });
+  };
```

Comment:
```suggestion
  handleIdInput = e => {
    this.setState({
      idInputValue: e.target.value,
    });
  };
```
이렇게 하면 좀 더 간편하죠? 
1. event 는 간단하게 e 로 표현할 수 있습니다.
2. 매개변수의 값이 하나라면 () 를 삭제해 주셔도 됩니다 :)
3. e.target.value 를 직접 바로 사용하셔도 됩니다!

Reply:
감사합니다 도현님!🙌